modules/installation-about-mirror-registry.adoc

```diff
@@ -7,18 +7,19 @@
+Whether using Red Hat Quay, Artifactory, Sonatype Nexus Repository, Harbor, or the mirror registry for Red Hat Openshift, all actions use the same process. The release image, which contains the description of the content, and the images it references are all mirrored. In addition, the Operator catalog source image and the images that it references must be mirrored for each Operator that you use. After you mirror the content, you configure each cluster to retrieve this content from your mirror registry.
+
+The mirror registry you choose can be any container registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2]. All major cloud provider registries, as well as Red Hat Quay, Artifactory, and others, have the necessary support. Using one of these registries ensures that {product-title} can verify the integrity of each image in disconnected environments.
 
 [IMPORTANT]
 ====
 The internal registry of the {product-title} cluster cannot be used as the target registry because it does not support pushing without a tag, which is required during the mirroring process.
 ====
 
+Your chosen mirror registry must be reachable by every machine in the clusters that you provision. If the registry is unreachable, installation, updating, or normal operations such as workload relocation might fail. For that reason, you must run mirror registries in a highly available way, and the mirror registries must at least match the production availability of your {product-title} clusters.
 
-The mirror registry must be reachable by every machine in the clusters that you provision. If the registry is unreachable installation, updating, or normal operations such as workload relocation might fail. For that reason, you must run mirror registries in a highly available way, and the mirror registries must at least match the production availability of your {product-title} clusters.
-
-When you populate a mirror registry with {product-title} images, you can follow two scenarios. If you have a host that can access both the internet and your mirror registry, but not your cluster nodes, you can directly mirror the content from that machine. This process is referred to as _connected mirroring_. If you have no such host, you must mirror the images to a file system and then bring that host or removable media into your restricted environment. This process is referred to as _disconnected mirroring_.
+When you populate your mirror registry with {product-title} images, you can follow two scenarios. If you have a host that can access both the internet and your mirror registry, but not your cluster nodes, you can directly mirror the content from that machine. This process is referred to as _connected mirroring_. If you have no such host, you must mirror the images to a file system and then bring that host or removable media into your restricted environment. This process is referred to as _disconnected mirroring_.
```

Comment:
This statement is confusing, as https://github.com/openshift/openshift-docs/pull/39593/files#diff-19a38c357f04d39f8deb746a2b85af0afce3745fbcb5a6605f2b8790449a97dfR16 states that OMR will not provide this. 

Thus it could be seen as OMR not meeting the 'defined requirements'. 
We somehow need to reconcile this, or make it clearer that 'this is a recommendation', for thing that are not OMR.